pom.xml

```diff
@@ -97,6 +97,16 @@
         <pax.cdi.version>0.12.0</pax.cdi.version>
         <validation.version>1.1.0.Final</validation.version>
         <commons-configuration.version>1.10</commons-configuration.version>
+        <servicemix.bundles.mina.version>1.1.7_6</servicemix.bundles.mina.version>
```

Comment:
Are all of these new version variables in the copy/pasted features file? If not, there is a good chance that they'll just become stale when we update camel again. Your best bet is to just copy and paste the features straight from the file with no modification. We should have comments in the features file stating as such.